client/blocks/blog-stickers/index.jsx

```diff
@@ -15,7 +15,7 @@ const BlogStickers = ( { blogId } ) => {
 
 	const { data: stickers } = useBlogStickersQuery( blogId );
 
-	if ( teams && ! isTeamMember ) {
+	if ( teams.length && ! isTeamMember ) {
```

Comment:
Necessary because `teams` will always be an array, therefore always will be truthy.